src/api/todos.ts

```diff
@@ -0,0 +1,20 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const USER_ID = 1621;
```

Comment:
Move this constant to another file (I'm not sure it's important, but my mentor said the same thing in a previous assignment)